generator/flog/random_test.go

```diff
@@ -1,12 +0,0 @@
-package flog
```

Comment:
this file had no tests, and was producing errors when I tried to run the tests locally, so I'm not sure what the goal of it is or how I'm supposed to run it?